includes/admin/meta-boxes/class-wcs-meta-box-subscription-data.php

```diff
@@ -185,7 +185,7 @@ public static function output( $subscription ) {
 							echo '<p><strong>' . esc_html( $field['label'] ) . ':</strong> ' . wp_kses_post( make_clickable( esc_html( $field_value ) ) ) . '</p>';
 						}
 
-						echo '<p' . ( ( '' != $subscription->get_payment_method() ) ? ' class="' . esc_attr( $subscription->get_payment_method() ) . '"' : '' ) . '><strong>' . esc_html__( 'Payment Method', 'woocommerce-subscriptions' ) . ':</strong>' . wp_kses_post( nl2br( $subscription->get_payment_method_to_display() ) ); // phpcs:ignore WordPress.PHP.StrictComparisons.LooseComparison
+						echo '<p' . ( ( '' != $subscription->get_payment_method() ) ? ' class="' . esc_attr( $subscription->get_payment_method() ) . '"' : '' ) . '><strong>' . esc_html__( 'Payment method', 'woocommerce-subscriptions' ) . ':</strong>' . wp_kses_post( nl2br( $subscription->get_payment_method_to_display() ) ); // phpcs:ignore WordPress.PHP.StrictComparisons.LooseComparison
```

Comment:
You'll notice as part of this PR that there are a couple of additional changes to capitalisation and removing `:`. This is so these labels are consistant with other labels in this section. 

eg 

| Before (title case) | After (sentence case) |
|--------|--------|
| <img width="331" alt="Screenshot 2023-10-31 at 1 25 22 pm" src="https://github.com/Automattic/woocommerce-subscriptions-core/assets/8490476/69346809-988e-4ffa-80bd-962d577a9404"> | <img width="326" alt="Screenshot 2023-10-31 at 1 26 46 pm" src="https://github.com/Automattic/woocommerce-subscriptions-core/assets/8490476/857cdcc2-9945-4dc4-a254-1b89b0451c45"> |



| Before | After |
|--------|--------|
| <img width="667" alt="Screenshot 2023-10-31 at 1 31 29 pm" src="https://github.com/Automattic/woocommerce-subscriptions-core/assets/8490476/1bb8c88a-fdeb-4b7c-84ad-c0ee6174b4e7"> <p align="center"><sup>Title case and erroneous `:`</p><sup> | <img width="684" alt="Screenshot 2023-10-31 at 1 29 19 pm" src="https://github.com/Automattic/woocommerce-subscriptions-core/assets/8490476/b508c48b-ef6d-4873-aa63-c19563e27a18"> |